docs/docs/30-installation/02-options.md

```diff
@@ -117,6 +117,12 @@ acorn install
     --service-lb-annotations service.beta.kubernetes.io/aws-load-balancer-nlb-target-type=instance
 ```
 
+Lastly, you can unset the annotations defined by the `--service-lb-annotations` flag by passing an empty annotation to the flag. For example:
+
+```bash
+acorn install --service-lb-annotations " "=" "
```

Comment:
I couldn't find anything immediately easier than this when working with maps. The rest of our CLI doesn't currently use maps as flags instead opting for parsing logic that runs against string flags. I'd love a better approach here if we can think of one.